stable/rookout/templates/deployment.yaml

```diff
@@ -23,6 +23,10 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          resources:
+          limits:
+            memory: "0.5Gi"
```

Comment:
I would say these as variables

Reply:
@cpanato what is a DCO and what do you mean by removing the chart version?